### ISSUE TITLE
[iOS] Clicking on the video during playback doesn't pause it

### ISSUE THREAD

**Issue by bjfletcher** [sampotts/plyr#1218]
### Expected behaviour

Clicking on the video during playback pauses it

### Actual behaviour

Clicking on the video during playback does *not* pause it

### Steps to reproduce

1. open webpage
2. click on the play icon to start the video
3. click on the video itself (not pause icon) to pause it

### Environment

Mobile Safari on iPhone and iPad

(it works ok on desktop Chrome)

### Link to where the bug is happening

https://plyr.io/

P.S. thank you for a really great video player!

**Comment by sampotts** [comment]
I think technically it should show controls ok the first tap and then while controls are visible then another tap should pause it. This is how most players work on touch devices from my experience anyway.

**Comment by bjfletcher** [comment]
That's fair enough. What do you think about the flicker (very brief, black block) that happens when you do the first tap? It is not very aesthetic, hey?

**Comment by sampotts** [comment]
I don't notice anything like that on my iPhone XS. What device are you testing on?

**Comment by BReyell** [comment]
I noticed the same issue today on both iOS 11 and iOS 12. Adding `pointer-events: none;` to `.plyr__poster` seems to have resolved the issue for me on iOS 10.

**Comment by bjfletcher** [comment]
> I don't notice anything like that on my iPhone XS. What device are you testing on?

iPhone XS Max running iOS 12.

@BReyell I've just tried that on iOS, no dice... I wonder if you meant that it works only on iOS 11?

**Comment by BReyell** [comment]
@bjfletcher Yeah, typo on my end. I meant it fixes iOS 11. iOS 12 is unchanged.

**Comment by effixx** [comment]
Also experiencing this issue.  Doesn't even show controls on first tap for me.

**Comment by ignacfetser** [comment]
i have just commented out the 'return' in plyr.js in this 'if block' and it solved the issue for me:
if (player.touch && player.config.hideControls) {
  //return;
}

**Comment by sampotts** [comment]
> i have just commented out the 'return' in plyr.js in this 'if block' and it solved the issue for me:
> if (player.touch && player.config.hideControls) {
> //return;
> }

That would make them always show I think?

**Comment by sampotts** [comment]
I've just tested the latest version of iOS (13.3.1) on https://plyr.io with HTML5, YouTube & Vimeo and the controls show on first tap. I agree about the large pause button like YouTube etc have to improve affordance. I'll look into that.

**Comment by nitaaikumar** [comment]
> I think technically it should show controls on the first tap and then while controls are visible then another tap should pause it. This is how most players work on touch devices from my experience anyway.

It is not pausing on double tap for me on iOS and Android. It is going fullscreen. Has this been implemented? If so, how do I make pausing on double tap work? Thanks for any help.

**Comment by trijammer** [comment]
> I've just tested the latest version of iOS (13.3.1) on https://plyr.io with HTML5, YouTube & Vimeo and the controls show on first tap. I agree about the large pause button like YouTube etc have to improve affordance. I'll look into that.

When using a Vimeo source, tapping on the video on iOS 14.4.1 doesn't show the controls at all. Only if captions are shown, then there's an area at the bottom of the video where the captions sit that work as a tap target, but tapping anywhere else does nothing.
